package.json

```diff
@@ -1,179 +1,188 @@
+        "graphql": "15.7.2",
+        "graphql-fields": "^2.0.3",
+        "graphql-request": "^4.2.0",
+        "graphql-scalars": "^1.10.0",
+        "handlebars": "^4.7.7",
+        "helmet": "^3.22.0",
+        "hot-shots": "^10.0.0",
+        "jsonwebtoken": "^9.0.0",
+        "keyv": "^4.0.3",
+        "lodash": "^4.17.21",
+        "log4js": "^6.4.0",
+        "moment": "^2.29.2",
+        "moment-timezone": "^0.5.35",
+        "multer": "^1.4.2",
+        "object-hash": "^3.0.0",
+        "pg": "^8.7.3",
+        "qrcode": "^1.4.4",
+        "reflect-metadata": "^0.1.13",
+        "source-map-support": "^0.5.21",
+        "talkjs": "^0.15.9",
+        "type-graphql": "^1.1.1",
+        "uuid": "^7.0.3",
+        "wildcard-match": "^5.1.2",
+        "ws": "^8.11.0"
     },
-    "transform": {
-      "^.+\\.[tj]sx?$": [
-        "ts-jest",
-        {
-          "useESM": true
-        }
-      ]
+    "devDependencies": {
+        "@types/basic-auth": "^1.1.3",
+        "@types/bcrypt": "^3.0.0",
+        "@types/cron": "^1.7.2",
+        "@types/ejs": "^3.0.5",
+        "@types/express": "^4.17.4",
+        "@types/graphql-fields": "^1.3.3",
+        "@types/jest": "^29.4.4",
+        "@types/jsonwebtoken": "^9.0.2",
+        "@types/keyv": "^3.1.2",
+        "@types/lodash": "^4.14.155",
+        "@types/mime-types": "^2.1.0",
+        "@types/multer": "^1.4.5",
+        "@types/node": "^12.12.31",
+        "@types/node-mailjet": "^3.3.3",
+        "@types/object-hash": "^3.0.2",
+        "@types/papaparse": "^5.0.3",
+        "@types/qrcode": "^1.5.1",
+        "@types/react-select": "^3.0.12",
+        "@types/uuid": "^8.3.4",
+        "@types/ws": "^8.5.3",
+        "@typescript-eslint/eslint-plugin": "^6.4.1",
+        "@typescript-eslint/parser": "^6.4.1",
+        "apidoc": "^0.22.1",
+        "concurrently": "^8.2.0",
+        "eslint": "^8.2.0",
+        "eslint-import-resolver-typescript": "^3.5.2",
+        "eslint-plugin-import": "^2.26.0",
+        "eslint-plugin-lernfair-lint": "./linter/",
+        "jest": "^29.5.0",
+        "lint-staged": "^14.0.1",
+        "nodemon": "^3.0.1",
+        "pre-commit": "^1.2.2",
+        "prettier": "^2.7.1",
+        "prisma": "4.15.0",
+        "ts-jest": "^29.0.5",
+        "typegraphql-prisma": "0.26.0",
+        "typescript": "^4.3.5"
     },
-    "collectCoverageFrom": [
-      "**/*.(t|j)s"
+    "optionalDependencies": {
+        "corona-school-matching": "1.3.0",
+        "html-pppdf": "^1.0.1"
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/corona-school/backend"
+    },
+    "keywords": [
+        "node",
+        "heroku",
+        "express"
     ],
-    "coverageDirectory": "../coverage",
-    "testEnvironment": "node"
-  },
-  "nodemonConfig": {
-    "verbose": true,
-    "ignore": [
-      ".git",
-      "node_modules/**/node_modules"
+    "license": "MIT",
+    "pre-commit": [
+        "lint-staged"
```

Comment:
Changed the precommit from our own bash script to the "lint-staged" package. The rest is just running prettier on all files